CHANGELOG.md

```diff
@@ -20,6 +20,8 @@
 
 - [File browser component][1677]. A basic file browser component has been
   implemented, but is not integrated into the user interface yet.
+- [Searcher icons][1715]. Some icons have been created that will be used in the
+  next version of the searcher.
```

Comment:
This are meant to be user product docs. Telling user "some icons have been created" doesn't really make any sense.